src/views/SystemProfiles/index.tsx

```diff
@@ -64,10 +64,7 @@ export const SystemProfiles = () => {
 
   const dispatch = useDispatch();
   useEffect(() => {
-    const getSystemIntakesEffect = async (): Promise<void> => {
-      dispatch(fetchSystemIntakes());
-    };
-    getSystemIntakesEffect();
+    dispatch(fetchSystemIntakes);
```

Comment:
Not concerned about this, since this view isn't exposed to the user. But this dispatch isn't invoked.

Reply:
good catch